src/Groups/matrices/matrix_manipulation.jl

```diff
@@ -138,6 +139,18 @@ end
 
 Return the permutation matrix over the ring `R` corresponding to the sequence `Q` or to the permutation `p`.
 If `Q` is a sequence, then `Q` must contain exactly once every integer from 1 to some `n`.
+
+#Examples
```

Comment:
```suggestion
# Examples
```

Reply:
@CameronBraunstein Unfortunately you did *not* fix this thoroughly in all applicable places *sigh*. I have now done this myself.